### PR TITLE
docs: fix some misspelling

### PR DESCRIPTION
**Pull request by eitsupi**
Related to #297

I ran `spelling::spell_check_package()` and fixed some typos.

### CODE REVIEW

**Review by etiennebacher — APPROVED** [review]
@eitsupi I don't know if you want to fix all typos here or do it gradually. I'm fine if you want to merge

File: DESCRIPTION [hunk 1]
```diff
@@ -28,6 +29,7 @@ Suggests:
     patrick,
     pillar,
     rmarkdown,
+    spelling,
```
Comment (eitsupi) [inline DESCRIPTION:32]: Is this needed here?
I believe `Config/Needs/dev` is more appropriate as it will only be used during development.

**Review by eitsupi — COMMENTED** [review]
Thank you for taking a look at this!
I did everything I could and am happy to merge this at this point.

File: DESCRIPTION (hunk 1 above)
Comment (etiennebacher) [reply DESCRIPTION]: indeed, that was an accidental commit

**Pull request merged**